pkg/blockchain/ratesAdmin_test.go

```diff
@@ -1,41 +1,45 @@
-package blockchain
+package blockchain_test
 
 import (
 	"context"
 	"math/big"
 	"testing"
-	"time"
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/stretchr/testify/require"
 	"github.com/xmtp/xmtpd/contracts/pkg/ratesmanager"
+	"github.com/xmtp/xmtpd/pkg/blockchain"
 	"github.com/xmtp/xmtpd/pkg/testutils"
+	"github.com/xmtp/xmtpd/pkg/testutils/anvil"
 )
 
-func buildRatesAdmin(t *testing.T) *RatesAdmin {
+func buildRatesAdmin(t *testing.T) (*blockchain.RatesAdmin, func()) {
 	ctx := context.Background()
 	logger := testutils.NewLog(t)
-	contractsOptions := testutils.GetContractsOptions(t)
+	rpcUrl, cleanup := anvil.StartAnvil(t, false)
+	contractsOptions := testutils.NewContractsOptions(rpcUrl)
+
 	// Set the nodes contract address to a random smart contract instead of the fixed deployment
-	contractsOptions.RatesManagerContractAddress = testutils.DeployRatesManagerContract(t)
+	contractsOptions.RatesManagerContractAddress = testutils.DeployRatesManagerContract(t, rpcUrl)
 
-	signer, err := NewPrivateKeySigner(
+	signer, err := blockchain.NewPrivateKeySigner(
 		testutils.LOCAL_PRIVATE_KEY,
 		contractsOptions.ChainID,
 	)
 	require.NoError(t, err)
 
-	client, err := NewClient(ctx, contractsOptions.RpcUrl)
+	client, err := blockchain.NewClient(ctx, contractsOptions.RpcUrl)
 	require.NoError(t, err)
 
-	ratesAdmin, err := NewRatesAdmin(logger, client, signer, contractsOptions)
+	ratesAdmin, err := blockchain.NewRatesAdmin(logger, client, signer, contractsOptions)
 	require.NoError(t, err)
 
-	return ratesAdmin
+	return ratesAdmin, cleanup
```

Comment:
if I understand `t.Cleanup` correctly, you dont have to return it here either.

Reply:
That's true. I was planning on doing a separate PR to replace all our cleanup functions with t.Cleanup...but I don't need to be adding new ones.